test/extended/authorization/podsecurity_admission.go

```diff
@@ -49,6 +51,105 @@ var _ = g.Describe("[sig-auth][Feature:PodSecurity]", func() {
 	})
 })
 
+var _ = g.Describe("[sig-auth][Feature:PodSecurity][Feature:SCC]", func() {
+	defer g.GinkgoRecover()
+
+	oc := exutil.NewCLIWithPodSecurityLevel("required-scc", psapi.LevelPrivileged)
+
+	g.It("required-scc annotation is being applied to workloads", func() {
+		privilegedSA, err := oc.AdminKubeClient().CoreV1().ServiceAccounts(oc.Namespace()).Create(
+			context.Background(),
+			&corev1.ServiceAccount{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "privileged-sa-",
+				},
+			},
+			metav1.CreateOptions{},
+		)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		sccRole, err := oc.AdminKubeClient().RbacV1().ClusterRoles().Create(context.Background(),
+			&v1.ClusterRole{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "required-scc-",
+				},
+				Rules: []v1.PolicyRule{
+					{APIGroups: []string{"security.openshift.io"}, Resources: []string{"securitycontextconstraints"}, Verbs: []string{"use"}},
+				},
+			}, metav1.CreateOptions{},
+		)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		_, err = oc.AdminKubeClient().RbacV1().RoleBindings(oc.Namespace()).Create(context.Background(),
+			&v1.RoleBinding{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "required-scc-",
+					Namespace:    oc.Namespace(),
+				},
+				RoleRef: v1.RoleRef{
+					APIGroup: "rbac.authorization.k8s.io",
+					Kind:     "ClusterRole",
+					Name:     sccRole.Name,
+				},
+				Subjects: []v1.Subject{
+					{
+						Kind:      "ServiceAccount",
+						Name:      privilegedSA.Name,
+						Namespace: oc.Namespace(),
+					},
```

Comment:
we actually don't need the SA for the test to succeed, correct?

Reply:
probably strictly not, as the admission logic is or'ing. i left it in for brevity though. not sure what permissions the requesting oc user has.

Reply:
I see, ok

Reply:
you know what, let me try 👍